src/OpenTelemetry.Resources.Azure/AzureResourceBuilderExtensions.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using OpenTelemetry.Internal;
+using OpenTelemetry.Resources.Azure;
+
+namespace OpenTelemetry.Resources;
+
+/// <summary>
+/// Extension methods to simplify registering of Azure resource detectors.
+/// </summary>
+public static class AzureResourceBuilderExtensions
+{
+    /// <summary>
+    /// Enables Azure AppService resource detector.
+    /// </summary>
+    /// <param name="builder"><see cref="ResourceBuilder" /> being configured.</param>
+    /// <returns>The instance of <see cref="ResourceBuilder" /> being configured.</returns>
+    public static ResourceBuilder AddAppServiceResourceDetector(this ResourceBuilder builder)
```

Comment:
```suggestion
    public static ResourceBuilder AddAppServiceDetector(this ResourceBuilder builder)
```

It can be nice if we can align the naming convention across detectors. Recent new extension methods have been introduced without `Resource` in the name like `AddHostDetector`, `AddProcessDetector` or `AddProcessRuntimeDetector`.